Makefile

```diff
@@ -1,11 +1,40 @@
+.PHONY: install
 install:
-	pip install --editable .
+	pip install -U pip setuptools
+	pip install -e .
+	pip install -r requirements.txt
+
+.PHONY: isort
+isort:
+	isort -rc -w 120 arq
+	isort -rc -w 120 tests
```

Comment:
Sorts imports bruh: https://github.com/timothycrosley/isort, for CI as well to ensure good looking code :)